server/db.json

```
@@ -0,0 +1,212 @@
+{
+  "waste": [
+    {
+      "id": "1301",
+      "description": "dropped icecream",
+      "typoeof": "rest",
```

```suggestion
      "typeof": "rest",
```
There seems to be a typo